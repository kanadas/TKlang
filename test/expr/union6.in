((\a b -> if a + b == 0 then 1 @ 'a' else a + b) :: int -> int -> (char + bool + int)) 1 2

