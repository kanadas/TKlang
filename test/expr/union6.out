expr/union6.in
Type mismatch: [(char+#2)] and [int] in expression:
if a + b == 0 then 1 @ 'a' else a + b
